next/components/forms/segments/AccountSections/HelpSection/HelpSection.tsx

```diff
@@ -14,9 +14,10 @@ const HelpSection = () => {
   return (
     <div className="flex flex-col">
       <AccountSectionHeader title={t('account_section_help.navigation')} />
-      <div className="w-full max-w-screen-lg mx-auto">
-        <h2 className="text-h2 justify-center hidden md:flex mt-8">Často kladené otázky</h2>
-        <div className="flex flex-col gap-2 md:gap-3 px-4 lg:px-0 my-4 md:my-6">
+      <div className="w-full max-w-screen-lg mx-auto py-6 lg:py-16">
+        <h2 className="text-h2 justify-start flex px-4 lg:px-0">Často kladené otázky</h2>
+        <div className="flex flex-col gap-2 md:gap-3 px-4 lg:px-0">
+          <h4 className="text-h4 justify-start flex mt-6">Všeobecné</h4>
```

Comment:
please ,when we're touching this, move these two hardoced texts into the translation file as well

Reply:
ohh sorry, forgot about that